qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
@anumahadevan : I would suggest to not change the default api version in this PR unless there is a strong reason to do so. 
We can do the version change in a separate PR.